Source/WebKit/Shared/API/Cocoa/WKRemoteObjectCoder.mm

```diff
@@ -458,16 +458,16 @@ static void encodeObject(WKRemoteObjectEncoder *encoder, id object)
     if (![object conformsToProtocol:@protocol(NSSecureCoding)] && ![object isKindOfClass:[NSInvocation class]])
         [NSException raise:NSInvalidArgumentException format:@"%@ does not conform to NSSecureCoding", object];
 
-    if (class_isMetaClass(object_getClass(object)))
+    if (class_isMetaClass(RetainPtr { object_getClass(object) }.get()))
```

Comment:
I know we need to do this to make the checker happy but adding retain/release here seems silly.

Reply:
Our convention elsewhere is that ObjC classes are considered immortal. As long as the return type of objc_getClass is accurate, we should be able to fix analysis to ignore this case. So I would suppress and file a bug for analysis.